publisher-sdk/src/main/java/com/criteo/publisher/CriteoBannerView.java

```diff
@@ -25,14 +27,15 @@
 import androidx.annotation.VisibleForTesting;
 import com.criteo.publisher.integration.Integration;
 import com.criteo.publisher.integration.IntegrationRegistry;
+import com.criteo.publisher.model.AdSize;
 import com.criteo.publisher.model.BannerAdUnit;
 
 public class CriteoBannerView extends WebView {
```

Comment:
What do you think about adding automated on the inflation from XML ?
You can take a look at `CriteoMediaViewTest` for instance.

Reply:
Sure, will add a test in another commit on this same PR.